backend/test/like_comment_handler_test.py

```diff
@@ -0,0 +1,152 @@
+        post_comment = self.post.comments.get(self.comment.id)
+        self.assertEquals(
+            len(post_comment.get('likes')), 1,
+            "This comment should have one like."
+        )
+        # assert the notification was sent
+        send_message_notification.assert_called_with(
+            self.third_user.key.urlsafe(),
+            self.user.key.urlsafe(),
+            "LIKE_COMMENT",
+            self.post.key.urlsafe(),
+            body.get('currentInstitution')
+        )
+
+        # Call the post method again
+        with self.assertRaises(Exception) as exc:
+            self.testapp.post_json(self.uri, body)
+        # Verify the exception message
+        exc = self.get_message_exception(exc.exception.message)
+        self.assertEquals(exc, "Error! User already liked this comment")
+        # Refresh post
+        self.post = self.post.key.get()
+        # get the comment
+        comment = self.post.get_comment(self.comment.id)
+        # Verify the number of likes in comment
+        self.assertEqual(
+            len(comment['likes']), 1,
+            "The number of likes should be 1, but it was %d" % len(comment['likes'])
+        )
+        
+    @patch('utils.verify_token', return_value={'email': 'otheruser@example.com'})
+    def test_delete(self, verify_token):
+        """Test the like_comment_handler's delete method."""
+        # like the comment
+        body = { "currentInstitution": {"name": "instName"} }
+        self.testapp.post_json(self.uri, body)
+        # Call the get method
+        data = self.testapp.get(self.uri)
+        # Get the body of request
+        likes = json.loads(data.body)
+        # assert the comment has one like
+        self.assertEquals(
+            len(likes), 1,
+            "The comment should have one like"
+        )
+        # call the delete method
+        self.testapp.delete(self.uri)
+        # Call the get method again
+        data = self.testapp.get(self.uri)
+        # Get the body of request
+        likes = json.loads(data.body)
+        # assert the comment has one like
+        self.assertEquals(
+            len(likes), 0,
+            "The comment should have no like"
+        )
+
+        # Call the delete method again
+        with self.assertRaises(Exception) as ex:
+            self.testapp.delete(self.uri)
+        # Verify exception message
+        ex = self.get_message_exception(ex.exception.message)
+        self.assertEquals(ex, "Error! User hasn't liked this comment.")
+        # Refresh the comment's likes
+        data = self.testapp.get(self.uri)
+        # Get the body of request
+        likes = json.loads(data.body)
+        # assert the comment has one like
+        self.assertEquals(
+            len(likes), 0,
+            "The number of likes should be zero, but it was %s" % len(likes)
+        )
+
+    def tearDown(cls):
+        """Deactivate the test."""
+        cls.test.deactivate()
+
+
+def init(cls):
```

Comment:
Remove init() and move the creation of mock to inside each test, only with the required entities.